.github/workflows/ios-build.yaml

```diff
@@ -0,0 +1,256 @@
+name: MacOS
```

Comment:
1. Can you rename it to avoid conflicts with the current MacOS github action?
2. headers :)